service/plugin/pom.xml

```diff
@@ -107,7 +107,12 @@
         <dependency>
             <groupId>com.solace</groupId>
             <artifactId>solace-messaging-client</artifactId>
-            <version>1.0.0</version>
+            <version>1.4.0</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-lang</groupId>
+            <artifactId>commons-lang</artifactId>
+            <version>2.6</version>
```

Comment:
Why are we using an older version?? 
https://mvnrepository.com/artifact/org.apache.commons/commons-lang3/3.16.0

Reply:
This is not an older version, but a different library. The one you mentioned is lang3 and the code has been using lang2. After up-versioning the `solace-messaging-client` library from 1.0.0 to 1.4.0 , it stopped bringing in the `commons-lang` dependency for plugins that are using it.

Since the amount of refactoring to completely remove lang2 was substantial and would change the content of this PR from 2 file change to 20+, I decided to not apply all those refactorings on this irrelevant PR.

Reply:
I understand , however, please note the older library has CVE vulnerability and we need to fix this

Reply:
I don't see any vulnerability reported here https://mvnrepository.com/artifact/commons-lang/commons-lang/2.6 , where did you see them?

An example of one `with` vulnerabilities https://mvnrepository.com/artifact/org.apache.commons/commons-lang3/3.2